packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -125,6 +184,7 @@ const metadata: Meta<WaferMapArgs> = {
             grid-max-y=${x => x.gridMaxY}
             :colorScale="${x => x.colorScale}"
             :dies="${x => getDiesSet(x.dies, wafermapDieSets)}"
+            :diesTable="${x => getDiesTableSet(x.diesTable, wafermapDiesTableSets)}"
```

Comment:
@atmgrifter00 setting the `diesTable` control does not change the `dies` control. How should I treat this use case? Should I hide the storybook controls until the feature is finished?

Reply:
I would presume these should be mutually exclusive, yes? If so, I think you should control which API you use through a Storybook control (like a switch). You can call the storybook option something like "die API", but it's just a boolean property that determines which property you bind to in your template code. You would then have a separate storybook property that represents the data set. This may be the option you call "dies", but it just presents the choice of data sets to use.

Reply:
I made a radio control for api versions